ios/Bridgeme.xcodeproj/xcshareddata/xcschemes/Yoda.xcscheme

```diff
@@ -66,7 +66,7 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
I don't know why this value must be changed into 'Release'.
But, I think it has a reason!
So, could you change this value into 'Release'?